blueocean-pipeline-editor/src/main/java/io/blueocean/rest/pipeline/editor/PipelineMetadataService.java

```diff
@@ -186,7 +184,7 @@ private boolean isWrapper(StepDescriptor d) {
 
     private boolean includeStep(StepDescriptor d) {
         boolean include = true;
-        if (ModelASTStep.getBlockedSteps().containsKey(d.getFunctionName())) {
+        if (BlockedStepsAndMethodCalls.blockedInSteps().containsKey(d.getFunctionName())) {
```

Comment:
Forced by some dependency update?

Reply:
Yeah upgrading to plugin 2.33 resulted in `@Restricted` violation coming from `ModelASTStep.getBlockedSteps()` required this change.